### PR TITLE
chore(deps): update to Angular 2.0.0

### PR DESCRIPTION
**Pull request by Nightapes**
update to Angular 2.0.0 Proprioception Reinforcement

### CODE REVIEW

**Comment by Martin-Luft** [comment]
But the final release is far, far away... https://github.com/angular/angular/milestones/

File: package.json [hunk 1]
```diff
@@ -53,19 +53,19 @@
...
-    "@angular/platform-browser": "2.0.0-rc.7",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.7",
-    "@angular/router": "3.0.0-rc.3",
+    "@angular/common": "2.0.0",
+    "@angular/compiler": "2.0.0",
+    "@angular/core": "2.0.0",
+    "@angular/forms": "2.0.0",
+    "@angular/platform-browser": "2.0.0",
+    "@angular/platform-browser-dynamic": "2.0.0",
+    "@angular/router": "3.0.0",
```
Comment (macav) [inline package.json:68]: would it make more sense to define it as `~2.0.0`, since there will be no more breaking changes within 2.0?

**Comment by Nightapes** [comment]
As it looks, it was the official release: http://angularjs.blogspot.de/2016/09/angular2-final.html

File: package.json (hunk 1 above)
Comment (Nightapes) [reply package.json:68]: Maybe we can wait till the next release, i'm not sure if there will be no more breaking changes.

Comment (valorkin) [reply package.json:68]: +1 to @Nightapes let it come a bit more stale

**Comment by valorkin** [comment]
@Martin-Wegner ng2 is far from perfect now, but it is already much better than year ago ;)

**Comment by valorkin** [comment]
@Nightapes Thanks for PR!

**Pull request merged**